import-export-cli/integration/testutils/environment_testUtils.go

```diff
@@ -124,5 +125,6 @@ func ValidateExportApiPassed(t *testing.T, args *ApiImportExportTestArgs, direct
 	t.Cleanup(func() {
 		//Remove Exported api
 		base.RemoveDir(directoryName)
+		base.WaitForIndexing()
```

Comment:
No need to wait here because we are only removing a directory and nothing related to indexing is taking place in APIM side.

Reply:
Fixed all the changes suggested with the latest commit